generated/nidaqmx/constants.py

```diff
@@ -57,8 +57,8 @@ class AccelUnits(Enum):
 
 
 class AcquisitionType(Enum):
-    FINITE = 10178  #: Acquire or generate a finite number of samples.
-    CONTINUOUS = 10123  #: Acquire or generate samples until you stop the task.
+    FINITE_SAMPS = 10178  #: Acquire or generate a finite number of samples.
+    CONT_SAMPS = 10123  #: Acquire or generate samples until you stop the task.
```

Comment:
This is going to nearly everything. I would prefer if these names didn't change.

Reply:
On further reflection, SAMPS doesn't repeat the enum name, so that's fine actually. But prefer SAMPLES

Reply:
Updated enum values as in expected constants.py (FINITE,CONTINUOUS)